src/accel_group.rs

```diff
@@ -0,0 +1,119 @@
+// Copyright 2019, The Gtk-rs Project Developers.
+// See the COPYRIGHT file at the top-level directory of this distribution.
+// Licensed under the MIT license, see the LICENSE file or <http://opensource.org/licenses/MIT>
+
+use gdk;
+use glib;
+use glib::object::{Cast, IsA};
+use glib::translate::*;
+use glib::ToValue;
+use gtk_sys;
+use AccelFlags;
+use AccelGroup;
+
+pub trait AccelGroupExtManual: 'static {
+    fn connect_accel_group<F>(
```

Comment:
maybe it need `#[must_use]`

Reply:
No, you don't *have* to use it. Only if you want to pass the `glib::Closure` to any of the other functions, like `disconnect()` :)

Reply:
Thanks for explanation